docs/src/pages/guides/minimizing-bundle-size/minimizing-bundle-size.md

```diff
@@ -12,14 +12,23 @@ Combined with [dangerJS](https://danger.systems/js/) we can inspect
 ## How to reduce the bundle size?
 
 For convenience, Material-UI exposes its full API on the top-level `material-ui` import.
-Using this is fine if you have tree shaking working,
-however, in the case where tree shaking is not supported or configured in your build chain, **this causes the entire library and its dependencies to be included** in your client bundle.
+If you're using ES 6 modules and a bundler that supports tree-shaking ([`webpack` >= 2.x](https://webpack.js.org/guides/tree-shaking/), [`parcel` with a flag](https://en.parceljs.org/cli.html#enable-experimental-scope-hoisting/tree-shaking-support)) you can safely
+use named imports and expect only a minimal set of Material-UI components in your bundle:
 
-You have couple of options to overcome this situation:
+```js
+import { Button, TextField } from '@material-ui/core';
+```
+
+Be aware that tree-shaking is an optimization that is usually only applied on production
```

Comment:
@mbrookes Just want to verify if this is the correct preposition. Reading this again it sounds like "applied to" is "more" correct. Maybe both work just as well?

Reply:
You're right – "applied to" sounds better.

Reply:
```suggestion
Be aware that tree-shaking is an optimization that is usually only applied to production
```